src/components/PhotoFrame.tsx

```diff
@@ -631,7 +633,7 @@ const PhotoFrame = ({
     return (
         <>
             {!isFirstLoad &&
-            files.length === 0 &&
+            files.length - incomingShareFiles.length === 0 &&
```

Comment:
can we just have `hasPersonalFiles` variable, so we wont as the value of this line. that would also simpligy this check line